db/mb_import.py

```diff
@@ -0,0 +1,33 @@
+import db
+import db.import_mb_data
+import logging
+from sqlalchemy import text
+import time
+
+SLEEP_DURATION = 30  # number of seconds to wait between runs
+
+
+def main():
+    logging.info("Checking if any import is required...")
```

Comment:
This message should be `musicbrainz_importer` started.

Reply:
:+1: